src/megabot-internals/inquirer.js

```diff
@@ -108,7 +108,7 @@ module.exports = {
               if (ids.emojis.reported) msg.addReaction(`${ids.emojis.reported.name}:${ids.emojis.reported.id}`)
               queue.createDeletionRequest(await zd.getSubmission(question.zd_id, ['users', 'topics']))
             }
-          }
+          } else if (emoji.id === ids.emoji.reported.id) return msg.removeReaction(`${ids.emojis.report.name}:${ids.emojis.reported.id}`, userID) // no one but us should add this emoji :angery:
```

Comment:
Please put this at the top of the block.

Reply:
What, don't you like wasting time checking every other type of reaction?

Reply:
While you're at it, the reported emoji isn't a guaranteed object in contrary to the other emotes
```suggestion
          } else if (ids,emojis.reported && emoji.id === ids.emoji.reported.id) return msg.removeReaction(`${ids.emojis.report.name}:${ids.emojis.reported.id}`, userID) // no one but us should add this emoji :angery:
```